src/bridge/bridgemain.h

```diff
@@ -810,6 +817,7 @@ typedef struct
     THREADPRIORITY Priority;
     THREADWAITREASON WaitReason;
     DWORD LastError;
+    NTSTATUS LastStatus;

```

Comment:
Same here, plugins will break with this change if you change this structure...

Reply:
Yep. I simply removed this field for now since it isn't used in the threads view (unlike LastError).